barretenberg/cpp/src/barretenberg/smt_verification/circuit/subcircuits.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "barretenberg/proof_system/circuit_builder/standard_circuit_builder.hpp"
+
+#include "barretenberg/smt_verification/circuit/circuit_schema.hpp"
+
+namespace smt_subcircuits {
+using namespace smt_circuit_schema;
+
+CircuitSchema get_standard_range_constraint_circuit(size_t n);
+CircuitSchema get_standard_logic_circuit(size_t n, bool is_xor);
```

Comment:
What is the purpose of these functions?

Reply:
They will be used in the further constraint system relaxation